ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiConfigManagementService.scala

```diff
@@ -60,19 +53,27 @@ private[apiserver] final class ApiConfigManagementService private (
     logger.info("Getting time model")
     index
       .lookupConfiguration()
-      .map(_.fold(defaultConfigResponse) { case (_, conf) => configToResponse(conf) })
+      .flatMap {
+        case Some((_, configuration)) =>
+          Future.successful(configurationToResponse(configuration))
+        case None =>
+          logger.warn(
+            "Could not get the current time model. The index does not yet have the ledger configuration."
```

Comment:
```suggestion
            "Could not get the current time model. The index does not yet have any ledger configuration."
```

There is no guarantee that `index.lookupConfiguration()` will return the "current" index. When restarting the index from a very old offset, it may take a long time until the indexer has ingested the latest config change. Same for the same warning message below.

Reply:
Good point, I'll improve the messaging.